screens/ActionsType.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigation, Link } from '@react-navigation/native';
+import {  TouchableOpacity} from 'react-native';
+import server from '../common/server';
+import { API, COLORS } from '../common/constants';
+import { useAppDispatch } from '../stores/hooks';
+import {Text, View,  Box, Radio, Button, Image, ScrollView } from 'native-base';
+import BackIcon from '../assets/images/back-icon.svg'
+import Checkbox from 'expo-checkbox';
+
+
+function ActionsType({route}) {
+  const {actionType} = route.params
+  const [results, setResults] = useState([])
+  const [selectedValues, setSelectedValues] = useState([]);
+  const [isChecked, setChecked] = useState(false);
+  const navigation = useNavigation()
+  const dispatch = useAppDispatch();
+
+
+  const getActionsByType = async (actionType: string) => {
+    if (!actionType) return;
+    const resp = await server.get(`${API.actions}/${actionType}`, {dispatch});
+    setResults(resp.data)
+    console.info('resp', JSON.stringify(resp.data));
+  }
+
+  const handleValueChange = (value) => {
+    if (selectedValues.some((selectedValue) => selectedValue.actionId === value.actionId)) {
+      setSelectedValues(selectedValues.filter((selectedValue) => selectedValue.actionId !== value.actionId));
+    } else {
+      setSelectedValues([...selectedValues, value]);
+    }
+  };
+  
+  const postAllActions = async () => {
+    let totalPoints = 0;
+    let totalUserPoints
+  
+    if (selectedValues) {
+      selectedValues.map( async (value, index) => {
+        await postAction(value.actionId);
+        totalPoints += value.actionPoints;
+        totalUserPoints = await addPoints(value.actionPoints);
+        console.log("points", totalUserPoints)
+
+        if (selectedValues.length === index + 1){
+          navigation.navigate("ActionsCongrats", {totalPoints, totalUserPoints})
+        }
+
+      });
+    }
+  };
+
+
+  const postAction = async (actionId: string) => {
+    const resp = await server.post(`${API.actions}/newaction`, {
+      actionId: actionId,
+
+    }, {dispatch})
+
```

Comment:
we are not doing anything with the response, is that intentional?

Reply:
Yes. Should I remove the const resp and leave only the await server.post... ?